ui/src/pages/Tenant/Detail/NewBackup/SchduleSelectFormItem.tsx

```diff
@@ -4,11 +4,12 @@ import type { FormInstance } from 'antd';
 import { Form, Space } from 'antd';
 import { clone } from 'lodash';
 import type { ParamsType } from './ScheduleSelectComp';
+import type { NewBackupForm, ScheduleDates } from '.';
```

Comment:
You had better declare these types in another file like `types.d.ts` instead of `index.tsx`